src/subcommand.rs

```diff
@@ -567,18 +585,24 @@ impl Subcommand {
               }
             }
 
-            print!(
-              "{list_prefix}{}",
-              RecipeSignature { name, recipe }.color_display(config.color.stdout())
-            );
-
-            format_doc(
-              config,
-              name,
-              doc.as_deref(),
-              max_signature_width,
-              &signature_widths,
-            );
+            if i == 0 || config.no_inline_aliases {
```

Comment:
I think if `no_inline_aliases` is false, i.e., we are printing aliases inline, we should modify the iterator above to not inline the aliases.

Reply:
Good call. I don't think I understood what the `iter::once().chain().enumerate()` was doing initially, but makes sense now!